src/YOUR_NAME_ex/pa02/chutes_simulation.py

```diff
@@ -0,0 +1,238 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Kaspar Akilles Lilja, Kevin Martin Lankut'
+__email__ = 'kalilja@nmbu.no, kela@nombu.no'
+
+
+import random
+
+
+class Board:
+
+    def __init__(self, chutes=None, ladders=None, goal=None):
+        """
+        :param chutes: a custom list of chutes
+        :param ladders: a custom list of ladders
+        :param goal: custom goal where you can choose the value
+        All these parameters have set standards, but can be altered.
+        """
+        if chutes is None:
+            chutes = [(24, 5), (33, 3), (42, 30), (56, 37), (64, 27),
+                      (74, 12), (87, 70)]
+
+        if ladders is None:
+            ladders = [(1, 40), (8, 10), (36, 52), (43, 62), (49, 79),
+                       (65, 82), (68, 85)]
```

Comment:
would recommend storing the default values outside the init as `_DEFAULT_CHUTES` or something. They will then be the same across all instances and they can be accessed by `self._DEFAULT_CHUTES`